pkg/build/controller/controller.go

```diff
@@ -172,7 +172,7 @@ func (bc *BuildController) nextBuildPhase(build *buildapi.Build) error {
 		build.Status.Reason = buildapi.StatusReasonCannotCreateBuildPod
 		return fmt.Errorf("failed to create build pod: %v", err)
 	}
-
+	build.Annotations[buildapi.BuildPodNameAnnotation] = podSpec.Name
```

Comment:
news to me, i would have expected it to always get initialized deeper in the api, but i guess i can initialize it.